operator/internal/configwatcher/configwatcher.go

```diff
@@ -46,6 +46,13 @@ type ConfigWatcher struct {
 	fs             afero.Fs
 	log            logr.Logger
 
+	// When deployed using operator cluster configuration is synced only
+	// in main reconciler. ConfigWatcher sidecar should only synchronize users
+	// and their passwords.
+	// Only when deployed via helm the ConfigWatcher must synchronize superuser
+	// cluster configuration list.
+	skipClusterConfigurationSync bool
```

Comment:
Same naming comment here, this should be scoped to just reference `superusers`

Reply:
I changed to variable name.